src/utils/hashPassword.ts

```diff
@@ -0,0 +1,29 @@
+export async function hashPasswordPBKDF2(password: string, salt: Uint8Array): Promise<string> {
```

Comment:
Where does this algorithm come from. Definitely falls for me in the 'don't roll your own crypto' bucket. Without a source or comments this would not pass a code review for me. I fully admit it's because I don't understand it fully.